migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc

```diff
@@ -9,12 +9,18 @@ toc::[]
 You can migrate your applications by using the {mtc-full} ({mtc-short}) web console or from the command line.
 
 include::modules/migration-prerequisites.adoc[leveloffset=+1]
+
+[discrete]
+=== Additional resources
+
+* link:https://docs.openshift.com/container-platform/3.11/install_config/registry/securing_and_exposing_registry.html#exposing-the-registry[Manually exposing a secure registry for {product-title} 3]
+
 include::modules/migration-updating-deprecated-internal-images.adoc[leveloffset=+2]
 include::modules/migration-creating-ca-bundle.adoc[leveloffset=+2]
 include::modules/migration-configuring-proxy-for-dvm.adoc[leveloffset=+2]
 include::modules/migration-writing-ansible-playbook-hook.adoc[leveloffset=+2]
 
-[id="migrating-applications-additional-resources_{context}"]
+[discrete]
 === Additional resources
```

Comment:
Please keep an ID attribute.  Even if it isn't a target for a link from the TOC, these common headings can be useful targets for tech support when they provide URLs to customers.

Some guidelines on the IDs: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
I believe the ID attribute is not required for discrete headers. See `migrating_from_ocp_3_to_4/planning-migration-3-4.adoc`, which is maintained by Andrea Hoffer.